.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--- ↑ Provide a general summary of your changes in the Title above ↑ -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests -->
+<!--- you ran to see how your change affects other areas of the -->
+<!--- code, etc. -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Stroke -->
+- [ ] Chore (non-breaking change which refactors / improves the existing code base)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to
+  change)
+
+## Checklist:
+
+<!--- If you're unsure about any of these, don't hesitate to ask. We're -->
+<!--- here to help! -->
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] My change requires a change to a package version.
+- [ ] I have updated the `package.json` version accordingly.
+- [ ] I have updated the `CHANGELOG.md` version accordingly.
+- [ ] I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
+
+[CONTRIBUTING_FILE]: https://github.com/fewlinesco/node-web-libraries/blob/master/CONTRIBUTING.md
```

Comment:
This points to an unrelated repo. I'd say we remove it for now, until we have a proper file for the monorepo.